vaccine_feed_ingest/runners/az/arcgis/normalize.py

```diff
@@ -145,13 +135,18 @@ def _normalize_hours(
     processed_hours = human_readable_hours
     if processed_hours is None:
         return []
+    processed_hours = processed_hours.upper()
 
     if processed_hours == "8-4":
         return [schema.OpenHour(day=day, opens="08:00", closes="16:00")]
-    if processed_hours == "8:00AM7:00PM":
-        return [schema.OpenHour(day=day, opens="08:00", closes="16:00")]
+    elif processed_hours == "8:00AM7:00PM":
+        return [schema.OpenHour(day=day, opens="08:00", closes="19:00")]
+    elif processed_hours.count("PM") == 2:
+        # "sat_hrs" for multiple Safeway locations are PM to PM, and
+        # "sun_hrs" is the same hours, but AM to PM.  just fix the typo.
+        processed_hours = processed_hours.replace("PM", "AM", 1)
```

Comment:
I'm a bit nervous that future updates to the dataset might include validly PM-PM times (e.g., `1:00PM - 8:00PM`).

Could you further limit this `if` to make sure we're updating only the locations with the typo?